sqlmesh/core/engine_adapter/base.py

```diff
@@ -1898,12 +1906,28 @@ def _build_view_properties_exp(
         properties: t.List[exp.Expression] = []
 
         if table_description:
-            properties.append(exp.SchemaCommentProperty(this=exp.Literal.string(table_description)))
+            properties.append(
+                exp.SchemaCommentProperty(
+                    this=exp.Literal.string(self._truncate_table_comment(table_description))
+                )
+            )
 
         if properties:
             return exp.Properties(expressions=properties)
         return None
 
+    def _truncate_comment(self, comment: str, length: t.Optional[int]) -> str:
+        if length is None:
```

Comment:
nit: you can make this a one-liner `return comment[:length] if length else comment`.